GetStream/stream-chat-swift
# Add default rendering for unsupported attachments

### 🔗 Issue Links
Resolves https://github.com/GetStream/ios-issues-tracking/issues/596

### 🎯 Goal
Add default rendering for unsupported attachments.

### 🎨 Showcase
<img width="250px" src="https://github.com/GetStream/stream-chat-swift/assets/12814114/a3de593c-2ddf-43c0-9978-d7495a842f82" />

### 🧪 Manual Testing Notes
1. Enable Location attachments in the Demo App
2. Add a location attachment to a channel
3. Disabled location attachments in the Demo App
4. Go back to the location attachment message
5. It should render as unsupported attachment

### ☑️ Contributor Checklist
